tests/fixtures/definitions/post.bp

```diff
@@ -2,6 +2,7 @@ models:
   Post:
     title: string
     author_id: id
+    author_bio: longtext
```

Comment:
I would have assumed this should be `longText` to emphasize the point. I've forgotten, does the underlying datatype get proper cased in the `Model`?

Reply:
inside `ModelLexer::$dataTypes` https://github.com/laravel-shift/blueprint/blob/031b8030605b8cf58dfc0e7940e43ce7f8c6d076/src/Lexers/ModelLexer.php#L32